docs/building.md

```diff
@@ -39,6 +39,45 @@ You can use `ligo` to also generate a contract initial storage, using the
 Following are the intructions to generate the initial storages of the DAO configuration
 provided in this codebase, see the [README](../README.md).
 
+## Generating a contract metadata
+
+BaseDAO metadata should be compliant with [TZIP-16](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md).
+
+
+To print the metadata you'll need the `stack` tool, see
+[The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/) tutorial
+for instructions on how to obtain it.
+
+```bash
+make metadata > metadata.json
+```
+
+This metadata should be stored either:
+- On IPFS: [Get started with IPFS](https://ipfs.io/)
+- As a separate contract: [The `tezos-storage` URI Scheme](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md#the-tezos-storage-uri-scheme)
+
+
+To use this metadata, follow [TZIP-16 Contract storage](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md#contract-storage)
+
+Specifically:
+> Unless otherwise-specified, the encoding of the values must be the direct stream
```

Comment:
I think we can remove the "Unless otherwise-specified" as we don't specify a different format here (now we should), so IMO we can just cut the quote to:
```suggestion
> the encoding of the values must be the direct stream
```

Reply:
Updated.